### PR TITLE
Add instance of bodyLimit to Route

### PR DESCRIPTION
**Pull request by i7326**
Add instance of bodyLimit to Route for changing the limit size of Body

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -44,10 +44,10 @@ const plugin = (fastify, opts, next) => {
 
     // populating pathRegex if missing
     route.pathRegex = undefined === route.pathRegex ? opts.pathRegex : String(route.pathRegex)
-
+    bodyLimit = (route.bodyLimit) ? {  bodyLimit: route.bodyLimit } : null;
```
Comment (jkyberneees) [inline index.js]: Not sure about the reason for this statement?

**Comment by jkyberneees** [comment]
Hi @i7326, thanks for reaching out with this PR. I fully agree on making this a new feature...
Would you please add the test cases for `bodyLimit` usage also?

NOTE: Please checkout my comment above. 

Thanks in advance.

File: index.js (hunk 1 above)
Comment (i7326) [reply index.js]: @jkyberneees I added that statement to keep the parameters clean, i actually missed something in first commit which later added to second commit. please have a look

**Comment by i7326** [comment]
sure, will add the test cases for bodyLimit

**Comment by jkyberneees** [comment]
Hi @i7326, thanks for your support and collaboration here. Merging and releasing ASAP.

**Comment by jkyberneees** [comment]
JFYI: https://github.com/jkyberneees/fastify-gateway/releases/tag/v2.6.0

**Pull request merged**